integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala

```diff
@@ -50,7 +48,7 @@ object CarbonThriftServer {
     val builder = SparkSession
```

Comment:
the 'storePath' parameter is not required, can remove it.

Reply:
fixed